pkg/cmd/pulumi/convert.go

```diff
@@ -120,34 +119,37 @@ func pclGenerateProject(directory string, project workspace.Project, p *pcl.Prog
 // pclEject
 func pclEject(directory string, loader schema.ReferenceLoader) (*workspace.Project, *pcl.Program, error) {
 	parser := hclsyntax.NewParser()
-	// Load all .pp files in the directory
-	err := filepath.WalkDir(directory, func(path string, d fs.DirEntry, err error) error {
-		if d.IsDir() {
-			return nil
+	// Load all .pp files in the components' subdirectory
```

Comment:
```suggestion
	// Load all .pp files in the directory
```
I think this comment can stay the same. It's just only loading the top level files now